testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -46,33 +46,13 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I select the addon "Legacy Module 12 x86_64"
     Then I should see the "Legacy Module 12 x86_64" selected
     # Comment following 1 line if you wish to re-enable testing with beta client tools for SLE12
-    And I deselect "SUSE Manager Client Tools Beta for SLE 12 x86_64 (BETA)" as a SUSE Manager product
+    # And I deselect "SUSE Manager Client Tools Beta for SLE 12 x86_64 (BETA)" as a SUSE Manager product
     When I click the Add Product button
     And I wait until I see "SUSE Linux Enterprise Server 12 SP5 x86_64" product has been added
     Then the SLE12 SP5 product should be added
 
 @scc_credentials
   Scenario: Add the initial product for the service pack migration
-    When I follow the left menu "Admin > Setup Wizard > Products"
-    And I wait until I do not see "Loading" text
-    And I enter "SUSE Linux Enterprise Server 15 SP2" as the filtered product description
-    And I wait until I see "SUSE Linux Enterprise Server 15 SP2 x86_64" text
-    And I open the sub-list of the product "SUSE Linux Enterprise Server 15 SP2 x86_64"
-    Then I should see a "Basesystem Module 15 SP2 x86_64" text
-    And I should see that the "Basesystem Module 15 SP2 x86_64" product is "recommended"
-    When I select "SUSE Linux Enterprise Server 15 SP2 x86_64" as a product
-    Then I should see the "SUSE Linux Enterprise Server 15 SP2 x86_64" selected
-    And I should see the "Basesystem Module 15 SP2 x86_64" selected
-    # Comment following 3 lines if you wish to re-enable testing with beta client tools for SLE15
-#    And I open the sub-list of the product "Basesystem Module 15 SP2 x86_64"
-#    And I open the sub-list of the product "SUSE Manager Client Tools for SLE 15 x86_64" on SUSE Manager
-#    And I deselect "SUSE Manager Client Tools Beta for SLE 15 x86_64 (BETA)" as a SUSE Manager product
-    When I click the Add Product button
-    And I wait until I see "SUSE Linux Enterprise Server 15 SP2 x86_64" product has been added
-    Then the SLE15 SP2 product should be added
-
-@scc_credentials
-  Scenario: Add a product with recommended enabled
     When I follow the left menu "Admin > Setup Wizard > Products"
```

Comment:
Seems duplicated to me with scenario `Enable "SUSE Linux Enterprise Desktop 15 SP3 x86_64" with recommended modules` from `testsuite/features/reposync/srv_enable_sync_products.feature`, unless I missed something.

(see previous remark)